pypeit/tests/test_pypeitsetup.py

```diff
@@ -168,3 +151,21 @@ def test_run_setup():
     os.remove(data_path('shane_kast_blue.sorted'))
 
 
+@dev_suite_required
```

Comment:
Thanks for adding this test. If I understand this is now a unit test that tries to run various operations on files known to have bad headers? Does this also test pypeit_obslog?

Reply:
yes, that is right